Makefile

```diff
@@ -29,7 +29,7 @@ kill:
 	$(DOCKER_COMPOSE_CMD) rm -f
 
 build: kill
-	COMPILE_ASSETS=true $(DOCKER_COMPOSE_CMD) build
+	COMPILE_ASSETS=true $(DOCKER_COMPOSE_CMD) build diet-error-handler publishing-e2e-tests $(APP_TO_BUILD)
```

Comment:
Do you mean to still have APP_TO_BUILD here? Also do we still need COMPILE_ASSETS?